### PR TITLE
Feature liq 883 pnl usd

### PR DESCRIPTION
**Pull request by PeGilo**

### CODE REVIEW

File: src/Lykke.Service.LiquidityEngine.AzureRepositories/CrossRateInstruments/CrossRateInstrumentEntity.cs [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+
+        public string QuoteSource
+        {
+            get => _quoteSource;
+            set
+            {
+                if (_quoteSource != value)
+                {
+                    _quoteSource = value;
+                    MarkValueTypePropertyAsDirty("QuoteSource");
```
Comment (aushakov) [inline src/Lykke.Service.LiquidityEngine.AzureRepositories/CrossRateInstruments/CrossRateInstrumentEntity.cs]: This method required only for value types

File: settings.yaml [hunk 2]
```diff
@@ -75,7 +75,7 @@ MatchingEngineClient:
       settings-key: MatchingEngine-Port
     Host:
       settings-key: MatchingEngine-Host
-
+
```
Comment (aushakov) [inline settings.yaml]: Exclude from changes

File: src/Lykke.Service.LiquidityEngine.AzureRepositories/Reports/SummaryReportEntity.cs [hunk 3]
```diff
@@ -72,6 +73,19 @@ public decimal PnL
...
 
+        public decimal? PnLUsd
+        {
+            get => _pnLUsd;
+            set
+            {
+                if (_pnLUsd != value)
+                {
+                    _pnLUsd = value;
+                    MarkValueTypePropertyAsDirty("PnLUsd");
```
Comment (aushakov) [inline src/Lykke.Service.LiquidityEngine.AzureRepositories/Reports/SummaryReportEntity.cs]: Only for value types

File: src/Lykke.Service.LiquidityEngine.AzureRepositories/CrossRateInstruments/CrossRateInstrumentEntity.cs (hunk 1 above)
Comment (PeGilo) [reply src/Lykke.Service.LiquidityEngine.AzureRepositories/CrossRateInstruments/CrossRateInstrumentEntity.cs]: Fixed.

File: settings.yaml (hunk 2 above)
Comment (PeGilo) [reply settings.yaml]: Fixed.

**Pull request merged**